tests/cases/compiler/declarationEmitParameterProperty2.ts

```diff
@@ -0,0 +1,5 @@
+// @strictNullChecks: true
```

Comment:
Can you swap this to

```suggestion
// @strictNullChecks: true,false
```

so that we can see the declaration emit for both. Ideally they should be the same.

Reply:
Looks like We need to change a few lines to make it work.